release/sdk-config.yaml.tmpl

```diff
@@ -50,6 +50,12 @@ commands:
   path: daml-helper/daml-helper
   desc: "Interact with a DAML ledger (experimental)"
   args: ["ledger"]
+- name: codegen
+  path: daml-helper/daml-helper
+  desc: "Run a language bindings code generation tool"
+  args: ["codegen"]
+- name: daml2ts
```

Comment:
Why do we need this? Two commands that do exactly the same thing mainly seems confusing.

Reply:
Is there a better way to do it? The first needs to exist in order that scala, java can be disambiguated from typescript. The second exists because it is called from `Run.hs` (the invocation of `daml2ts` needs to run in a DAML env). Note the second is "hidden".

Reply:
You should be calling the `daml2ts` binary directly from `runDaml2ts` instead of invoking the assistant again.

Reply:
This is the connection (`Run.hs`):
```
runDaml2ts :: [String] -> IO ()
runDaml2ts remainingArgs = do
    -- Call back to the assistant because this needs to run in a DAML
    -- env.
    procConfig <- toAssistantCommand $ ["daml2ts"] ++ remainingArgs
    runProcess_ procConfig
```
(called from `Main.hs`).

Reply:
You can get the location of the actual binary with something like `fmap (</> "daml2ts" </> "daml2ts") getSdkPath`.

Reply:
> You should be calling the `daml2ts` binary directly from `runDaml2ts` instead of invoking the assistant again.

Should I? Can you help out with an example of how (respecting that it needs to run in the DAML env)? I really thought I was doing this as it was designed to be used and my meeting with @associahedron  seemed to confirm that (or at least he didn't tell me it was wrong).

Reply:
See above for how you can get the path to the binary and then just run that. I don’t think adding hidden commands makes any sense so that you can proxy back through the assistant makes sense.

Reply:
Thanks for the help. All fixed!